perfkitbenchmarker/linux_benchmarks/mysql_service_benchmark.py

```diff
@@ -558,6 +572,21 @@ def Prepare(self, vm):
     logging.info('Successfully created an RDS DB instance. Address is %s',
                  vm.db_instance_address)
     logging.info('Complete output is:\n %s', response)
+     
+  def _validate_size(self, size):
```

Comment:
For some reason, PKB's style is CamelCase for function names, so please change it to be consistent with the rest of the file. You can leave the underscore.